### PR TITLE
Label boundaries and potential owners addition

### PR DESCRIPTION
**Pull request by benmichel**
- Add Label boundaries when missing on BPMNShapes and BPMNEdges
- Add PotentialOwners on UserTasks

### CODE REVIEW

**Comment by benmichel** [comment]
- Schema location has been reverted to is original value
- Unwanted CDatas hav been removed from camunda extensionElements

File: Reference/C.1.0.bpmn [hunk 1]
```diff
@@ -387,20 +546,30 @@
...
+          <w4graph:basic background="190,0,0" foreground="0,0,0" autoResize="false" borderColor="100,100,100" collapsed="false"/>
+        </w4graph:graphStyle>
       </extensionElements>
       <incoming>SequenceFlow_3</incoming>
     </endEvent>
-    <serviceTask id="archiveInvoice" camunda:delegateExpression="#{archiveService}" name="Archive&#xA;Invoice" implementation="webService">
+    <serviceTask id="archiveInvoice" camunda:delegateExpression="#{archiveService}" name="Archive&#xA;Invoice" completionQuantity="1" isForCompensation="false" startQuantity="1" implementation="webService">
       <extensionElements>
         <camunda:field name="text0" stringValue="Hello World"/>
+        <camunda:field name="text1"><![CDATA[
```
Comment (falko) [inline Reference/C.1.0.bpmn]: This additional line breaks the XML structure.

Comment (falko) [inline Reference/C.1.0.bpmn]: @benmichel I'll remove the line manually to get the issue closed, but please fix your tool as well.

**Comment by benmichel** [comment]
Hi falko,

This must be a github merge issues because the file i submitted (present on my file system) does not contains this line...

**Comment by falko** [comment]
Okay, even more a reason to fix it manually.

**Comment by falko** [comment]
With that fix the model still executes fine in Camunda BPM.

**Pull request merged**